network-api/networkapi/wagtailpages/translation.py

```diff
@@ -114,11 +113,6 @@ class ParticipatePage2TR(TranslationOptions):
     fields = ()
 
 
-@register(PeoplePage)
```

Comment:
keep this, but move it to the end of the file, with another comment that explains that everything below the comment is obsolete code and will need to be cleaned up in the future.